src/Requests/Issue/IssueRequest.php

```diff
@@ -191,4 +191,117 @@ public function notify($issueIdOrKey)
     {
         // TODO: implement
     }
+
+    /**
+     * Returns all worklogs for an issue.
+     *
+     * @see https://developer.atlassian.com/cloud/jira/platform/rest/v3#api-rest-api-3-issue-issueIdOrKey-worklog-get
+     *
+     * @param string|int $issueIdOrKey
+     * @param \Atlassian\JiraRest\Requests\Issue\Parameters\GetParameters|array $parameters
+     *
+     * @return \GuzzleHttp\Psr7\Response
+     * @throws \Atlassian\JiraRest\Exceptions\JiraClientException
+     * @throws \Atlassian\JiraRest\Exceptions\JiraNotFoundException
+     * @throws \Atlassian\JiraRest\Exceptions\JiraUnauthorizedException
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws \TypeError
+     */
+    public function getWorklogs($issueIdOrKey, $parameters = [])
+    {
+        $this->validateParameters($parameters, SearchParameters::class);
+
+        return $this->execute('get', "issue/{$issueIdOrKey}/worklog", $parameters);
+    }
+
+    /**
+     * Returns all worklogs for an issue.
+     *
+     * @see https://developer.atlassian.com/cloud/jira/platform/rest/v3#api-rest-api-3-issue-issueIdOrKey-worklog-get
+     *
+     * @param string|int $issueIdOrKey
+     * @param int $workdlogId
+     * @param \Atlassian\JiraRest\Requests\Issue\Parameters\GetParameters|array $parameters
```

Comment:
The documentation does not line up with the validation https://github.com/rjvandoesburg/laravel-jira-rest-client/pull/11/files#diff-fd7619950db52bec2613184c9a1032c4R235

This is the same for all methods

Reply:
You mean should it be the other way around? Ok what about now